glossary.yml

```
@@ -588,13 +619,26 @@
     def: >
       FIXME
 
+- slug: "false"
+  en: 
+    term: "false"
+    def: > 
+       The logical (Boolean) state opposite of "true". Used in logic and programming to represent binary state of something. 
+       Often fluidly represented as true/false, on/off, 1/0. 
+    ref:
+     - boolean
+     - true
+     - truthy
+     - falsy 
+
 - slug: falsy
   en:
     term: "falsy"
     def: >
-      A horrible neologism meaning "equivalent to false".
+      Evaluating to a state of false in a Boolean context. 
```

```suggestion
      Evaluating to [false](#false) in a [Boolean](#boolean) context. 
```

Making this consistent with the true definition